crates/sui-framework/packages/sui-framework/sources/bridge/bridge.move

```diff
@@ -0,0 +1,363 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#[allow(unused_const)]
+module sui::bridge {
+    use std::vector;
+
+    use sui::address;
+    use sui::balance;
+    use sui::bcs;
+    use sui::bridge_committee::{Self, BridgeCommittee};
+    use sui::bridge_escrow::{Self, BridgeEscrow};
+    use sui::bridge_treasury::{Self, BridgeTreasury, token_id};
+    use sui::chain_ids;
+    use sui::coin::{Self, Coin};
+    use sui::event::emit;
+    use sui::object::{Self, UID};
+    use sui::table::{Self, Table};
+    use sui::transfer;
+    use sui::tx_context::{Self, TxContext};
+
+    #[test_only]
+    use sui::bridge_treasury::USDC;
+    #[test_only]
+    use sui::hex;
+    #[test_only]
+    use sui::test_scenario;
+
+    struct Bridge has key {
```

Comment:
Have we considered making this object versioned and upgradeable, just like system state obj and random obj?